services/movie.py

```diff
@@ -15,18 +16,22 @@ def get_movies(
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__icontains=title)
+
     return queryset
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
```

Comment:
Consider handling the `DoesNotExist` exception that can be raised by `Movie.objects.get(id=movie_id)`. This will prevent runtime errors if the movie ID is not found.